src/main/java/org/jabref/gui/openoffice/OOBibBase.java

```diff
@@ -618,12 +641,7 @@ public void guiActionInsertEntry(List<BibEntry> entries,
             OOError.from(ex).setTitle(errorTitle).showErrorDialog(dialogService);
         } catch (DisposedException ex) {
             OOError.from(ex).setTitle(errorTitle).showErrorDialog(dialogService);
-        } catch (CreationException
-                 | WrappedTargetException
-                 | IOException
-                 | PropertyVetoException
-                 | IllegalTypeException
-                 | NotRemoveableException ex) {
+        } catch (Exception ex) {
```

Comment:
Never catch all exceptions

Reply:
Check agian @subhramit  and other places

Reply:
Done